charts/puris-backend/values.yaml

```diff
@@ -0,0 +1,129 @@
+# Default values for puris-backend.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicaCount: 1
+
+image:
+  repository: ghcr.io/catenax-ng/tx-puris-backend/puris-backend
+  pullPolicy: IfNotPresent
+  # Overrides the image tag whose default is the chart appVersion.
+  #tag: "latest"
```

Comment:
Provide image tag as empty key `tag: ""` instead of commenting it out.

Reply:
In a first version, I tried using an empty tag which results in `Error: INSTALLATION FAILED: unable to build kubernetes objects from release manifest: error validating "": error validating data: apiVersion not set`.

I was able to fix the bug that allows me to follow your guidelines and use an empty tag.